### PR TITLE
Bug 2060499: Template additional resources

### PR DESCRIPTION
**Pull request by upalatucci**
## 📝 Description

Show which objects will be created alongside the VM

## 🎥 Demo

Screenshot from 2022-08-02 11-40-49
Screenshot from 2022-08-02 11-40-17

### CODE REVIEW

File: src/utils/components/AdditionalResources/AdditionalResources.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  const { t } = useKubevirtTranslation();
+
+  if (template.objects.length === 1) return null;
+
+  return (
+    <DescriptionListGroup>
+      <DescriptionListTerm>{t('Additional resources')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        <ul>
+          {template.objects
```
Comment (metalice) [inline src/utils/components/AdditionalResources/AdditionalResources.tsx:26]: please add ?. , there is quite a few in this file :)

**Review by metalice — COMMENTED** [review]
Hi @upalatucci  , nice , left only 1 small comment :)

File: src/utils/components/AdditionalResources/AdditionalResources.tsx (hunk 1 above)
Comment (upalatucci) [reply src/utils/components/AdditionalResources/AdditionalResources.tsx]: solved thanks

**Comment by upalatucci** [comment]
thanks @metalice, fixed

**Review by hstastna — APPROVED** [review]
Nice work, Ugo! I have nothing to add 👍🏽

File: src/utils/components/AdditionalResources/AdditionalResources.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+} from '@patternfly/react-core';
+
+type AdditionalResourcesProps = {
+  template: V1Template;
+};
+
+const AdditionalResources: React.FC<AdditionalResourcesProps> = ({ template }) => {
+  const { t } = useKubevirtTranslation();
+
+  if (!template?.objects || template?.objects.length === 1) return null;
```
Comment (metalice) [inline src/utils/components/AdditionalResources/AdditionalResources.tsx:19]: IsEmpty(template?.objects) can be used here :)

File: src/utils/components/AdditionalResources/AdditionalResources.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  return (
+    <DescriptionListGroup>
+      <DescriptionListTerm>{t('Additional resources')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        <ul>
+          {template?.objects
+            ?.filter((object) => object?.kind && object.kind !== VirtualMachineModel.kind)
```
Comment (metalice) [inline src/utils/components/AdditionalResources/AdditionalResources.tsx:27]: VirtualMachineModel.kind is guaranteed as its hardcoded, so compare of object?.kind !== VirtualMachineModel.kind , is sufficient here i guess. :)

File: src/utils/components/AdditionalResources/AdditionalResources.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            .map((object) => (
+              <li key={`${object.kind}-${object?.metadata?.name}`}>{object.kind}</li>
```
Comment (metalice) [inline src/utils/components/AdditionalResources/AdditionalResources.tsx:29]: ?.kind

**Pull request merged**